WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.java

```diff
@@ -150,52 +130,64 @@ public void onSaveInstanceState(Bundle outState) {
-        if (mPostList != null) {
-            mPostList.onPositiveClickedForBasicDialog(instanceTag);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onPositiveClickedForBasicDialog(instanceTag);
         }
     }
 
     @Override
     public void onNegativeClicked(@NotNull String instanceTag) {
-        if (mPostList != null) {
-            mPostList.onNegativeClickedForBasicDialog(instanceTag);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onNegativeClickedForBasicDialog(instanceTag);
         }
     }
 
     @Override
     public void onDismissByOutsideTouch(@NotNull String instanceTag) {
-        if (mPostList != null) {
-            mPostList.onDismissByOutsideTouchForBasicDialog(instanceTag);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onDismissByOutsideTouchForBasicDialog(instanceTag);
         }
     }
 
-    // GutenbergWarningDialogClickInterface Callbacks
+// GutenbergWarningDialogClickInterface Callbacks
 
     @Override
     public void onGutenbergWarningDialogEditPostClicked(long gutenbergRemotePostId) {
-        if (mPostList != null) {
-            mPostList.onGutenbergWarningDialogEditPostClicked(gutenbergRemotePostId);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onGutenbergWarningDialogEditPostClicked(gutenbergRemotePostId);
         }
     }
 
     @Override
     public void onGutenbergWarningDialogCancelClicked(long gutenbergRemotePostId) {
-        if (mPostList != null) {
-            mPostList.onGutenbergWarningDialogCancelClicked(gutenbergRemotePostId);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onGutenbergWarningDialogCancelClicked(gutenbergRemotePostId);
         }
     }
 
     @Override
     public void onGutenbergWarningDialogLearnMoreLinkClicked(long gutenbergRemotePostId) {
-        if (mPostList != null) {
-            mPostList.onGutenbergWarningDialogLearnMoreLinkClicked(gutenbergRemotePostId);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onGutenbergWarningDialogLearnMoreLinkClicked(gutenbergRemotePostId);
         }
     }
 
     @Override
     public void onGutenbergWarningDialogDontShowAgainClicked(long gutenbergRemotePostId, boolean checked) {
-        if (mPostList != null) {
-            mPostList.onGutenbergWarningDialogDontShowAgainClicked(gutenbergRemotePostId, checked);
+        PostListFragment fragment = getCurrentFragment();
+        if (fragment != null) {
+            getCurrentFragment().onGutenbergWarningDialogDontShowAgainClicked(gutenbergRemotePostId, checked);
         }
     }
+
+    @Nullable
```

Comment:
I am not sure, but I _think_ we prefer these annotations inline as opposed to annotations like `@Override`. Should we change the following line to:

```
private @Nullable PostListFragment getCurrentFragment() {
```

Reply:
Fixed in [39a5ad4](https://github.com/wordpress-mobile/WordPress-Android/pull/9226/commits/39a5ad4ce6e3530c096b11a24ae5fcc5e804f5a4)